### DISCUSSION TITLE
Windows key remapping

### DISCUSSION THREAD

**Discussion by rp1231** [houmain/keymapper#144] [Q&A]
Is it possible to remap the windows key with keymapper?

**Comment by ristomatti** [comment]
I'm almost certain you can, but I don't have Windows so I can't say for sure. There's only one way to find out...

**Reply by ristomatti** [reply]
Press "Win" on this page linked on the README https://www.toptal.com/developers/keycode and you'll find out what key name to use in your config:

```ini
# For convenience 
Win = MetaLeft

# Open a text editor and see if this works
Win{T} >> 'test'
```

**Reply by rp1231** [reply]
@ristomatti I found out that it is possible to remap the windows key with remapper, which is why I closed the discussion.
I should have probably mentioned why I closed the discussion.
Appreciate the reply though.
Thanks

**Reply by ristomatti** [reply]
Ah sorry, didn't notice. :+1:
